server/imports/core/connection/helper.js

```diff
@@ -32,27 +32,36 @@ ConnectionHelper.prototype = {
 
   putCorrectDBToConnectionUrl(connection) {
     let options = '';
-    if (connection.url.indexOf('?') !== -1) options = `?${connection.url.split('?')[1]}`;
+    if (connection.url.indexOf('?') !== -1) {
+      options = `?${connection.url.split('?')[1]}`;
+    }
 
     const splited = connection.url.split('/');
-    if (splited.length <= 3) connection.url += `/${connection.databaseName}${options}`;
-    else {
+    if (splited.length <= 3) {
+      connection.url += `/${connection.databaseName}${options}`;
+    } else {
       splited[3] = '';
-      connection.url = splited.join('/') + connection.databaseName + options;
+      connection.url =
```

Comment:
Codacy has a fix for the issue: [There should be no line break before or after '='.](https://app.codacy.com/gh/nosqlclient/nosqlclient/pullRequest?prid=7730964)
```suggestion
      connection.url =        splited.slice(0, 3).join('/') + `/${connection.databaseName}${options}`;
```